mvc/3.1/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Jakarta MVC 3.1 (Under Development)"
+date: 2025-04-15
+summary: "Release supporing Jakarta EE 12"
+---
+
+Jakarta MVC defines a standard for creating web applications following the action-based model-view-controller pattern.
+
+This release ensures alignment with Jakarta RESTful Web Services **TODO version**.
```

Comment:
Should this be version 5.0 to align with https://github.com/jakartaee/specifications/pull/823 or is that not yet decided?

Reply:
5.0